.gitignore

```diff
@@ -28,6 +28,7 @@ bld/
 .vs/
 # Uncomment if you have tasks that create the project's static files in wwwroot
 #wwwroot/
+Migrations/
```

Comment:
Ignoring the `Migrations` folder looks like it was a mistake here. Important stuff goes in that folder.